src/crypto-api/index.ts

```diff
@@ -364,6 +365,22 @@ export interface CryptoApi {
      */
     getEncryptionInfoForEvent(event: MatrixEvent): Promise<EventEncryptionInfo | null>;
 
+    /**
+     * Encrypts a given payload object via Olm to-device messages to a given
+     * set of devices.
+     *
+     * @param eventType the type of the event to send
```

Comment:
doc-comments in the js-sdk put a `-` between param name and description. (I actually thought we enforced this with a lint, but seems I'm wrong.)

They also end in a full-stop, preferably, though there are plenty of counterexamples of that.

```suggestion
     * @param eventType - the type of the event to send.
```

likewise elsewhere